### DISCUSSION TITLE
GitHub is not referencing index.html

### DISCUSSION THREAD

**Discussion by mathcat4** [community/community#23961] [Other Feature Feedback, Questions, & Ideas]
<p>I am using git hub pages for <a href="https://github.com/Tams-Tams/tams-tams.github.io" rel="noopener nofollow ugc">Javascript Files.</a>. I followed the Github Pages Tutorial made from Github, and the VideoMOtion Page is Working, but the root directory is not working , even with the index.html page.</p>
<p>What did I do wrong, help me please?<br>
Tams</p>

**Comment by otter-computer** [comment, marked as answer]
<p>It looks like your site is loading for me now, is it doing the same for you? It can take up to 10 minutes to finish building your site on GitHub, and another 10 to propagate across the CDN, so you may have to wait after enabling Pages or pushing changes to see your site.</p>

**Comment by mathcat4** [comment]
<p>Thanks, I just had to wait.</p>
